prime-router/src/main/kotlin/FakeReport.kt

```diff
@@ -109,6 +109,25 @@ class FakeDataService : Logging {
             return "${context.patientName.username()}@email.com"
         }
 
+        // ID values typically come from valuesets, so we pass in the element
+        // and then we examine both the alt values and the value set for the element
+        // and we then pull a random value from all the potential values available.
+        // the rationale here is that if an element specifies alt values, they are preferred
+        // to the values held in the value set collection, and so we use them first
+        fun createFakeValueFromValueSet(element: Element): String {
+            val altValues = element.altValues
+            val valueSet = element.valueSetRef
+            // if the code defines alternate values in the schema we need to
+            // output them here
+            val possibleValues = if (altValues?.isNotEmpty() == true) {
+                altValues.map { it.code }.toTypedArray()
+            } else {
+                valueSet?.values?.map { it.code }?.toTypedArray() ?: arrayOf("")
```

Comment:
Maybe a quick inline comment why a blank string, or maybe mention that in the kdoc.  Reason I ask is that the code in line 217 is using ifEmpty() and an array("") will not be empty.